frontend/src/calculations/FieldAndSoil/Crops/Calculations.ts

```diff
@@ -580,3 +595,125 @@ export async function getBlueberryNutrients(
+    if (selectedCrop.id === CROP_RASPBERRIES_ID) {
+      const nutrients = await getRaspberryNutrients(
+        cropDataForCalc.yield,
+        cropDataForCalc.willSawdustBeApplied!,
+        cropDataForCalc.willPlantsBePruned!,
+        cropDataForCalc.whereWillPruningsGo!,
+        field.soilTest?.valP !== undefined
+          ? field.soilTest.valP
+          : DEFAULT_BERRY_DATA.defaultRaspberrySoilTestP,
+        field.soilTest?.valK !== undefined
+          ? field.soilTest.valK
+          : DEFAULT_BERRY_DATA.defaultRaspberrySoilTestK,
+        cropDataForCalc.leafTissueP !== undefined
+          ? cropDataForCalc.leafTissueP
+          : DEFAULT_BERRY_DATA.defaultRaspberryLeafTestP,
+        cropDataForCalc.leafTissueK !== undefined
+          ? cropDataForCalc.leafTissueK
+          : DEFAULT_BERRY_DATA.defaultRaspberryLeafTestK,
+      );
+      nutrientValues = extractNutrientValues(nutrients);
+    } else if (selectedCrop.id === CROP_BLUEBERRIES_ID) {
+      const nutrients = await getBlueberryNutrients(
+        cropDataForCalc.yield,
+        cropDataForCalc.willSawdustBeApplied!,
+        cropDataForCalc.willPlantsBePruned!,
+        cropDataForCalc.whereWillPruningsGo!,
+        cropDataForCalc.plantAgeYears!,
+        cropDataForCalc.numberOfPlantsPerAcre!,
+        field.soilTest?.valP !== undefined
+          ? field.soilTest.valP
+          : DEFAULT_BERRY_DATA.defaultBlueberrySoilTestP,
+        cropDataForCalc.leafTissueP !== undefined
+          ? cropDataForCalc.leafTissueP
+          : DEFAULT_BERRY_DATA.defaultBlueberryLeafTestP,
+        cropDataForCalc.leafTissueK !== undefined
+          ? cropDataForCalc.leafTissueK
+          : DEFAULT_BERRY_DATA.defaultBlueberryLeafTestK,
+      );
+      nutrientValues = extractNutrientValues(nutrients);
+    } else {
+      // Calculate crop requirements (P2O5, K2O, N)
+      const cropRequirementN = getCropRequirementN(cropDataForCalc, selectedCrop, selectedCropType);
+      const cropRequirementP205 = await getCropRequirementP205(
+        cropDataForCalc,
+        field.soilTest,
+        farmRegion,
+      );
+      const cropRequirementK2O = await getCropRequirementK2O(
+        cropDataForCalc,
+        field.soilTest,
+        farmRegion,
+      );
+
+      // Calculate crop removals (N, P2O5, K2O)
+      const cropRemovalN = getCropRemovalN(cropDataForCalc, selectedCrop, selectedCropType);
+      const cropRemovalP205 = getCropRemovalP205(cropDataForCalc, selectedCrop, selectedCropType);
+      const cropRemovalK20 = getCropRemovalK20(cropDataForCalc, selectedCrop, selectedCropType);
+
+      nutrientValues = {
+        cropRequirementN,
+        cropRequirementP205,
+        cropRequirementK2O,
+        cropRemovalN,
+        cropRemovalP205,
+        cropRemovalK20,
+      };
+    }
+
+    // Mark calculations as performed
+    return nutrientValues;
+  } catch (error) {
+    console.error('Error calculating crop data:', error);
+  }
+  return false;
```

Comment:
Why is false returned here? It seems like the goal is to let the calling function know an error was thrown, but if that's the case, you should probably just not catch the error and let the calling function catch and handle it.

Reply:
Eslint demands a default return. Error are caught by the catch, but eslint doesn't care. Should I write `//eslint ignore` instead?